Docs/ProjectSpec.md

```diff
@@ -580,21 +581,24 @@ This is used to override settings or run build scripts in specific targets
 
 This is a template that can be referenced from a normal target using the `templates` property. The properties of this template are the same as a [Target](#target)].
 Any instances of `$target_name` within each template will be replaced by the final target name which references the template.
+Attributes defined in `templateAttributes` will be replaced by the value specified in the target which references the template.
```

Comment:
The `templateAttributes` are defined in the target, not the template. This sentence doesn't make that so clear.

Reply:
I'm not a native speaker. Do you have a suggestion for a better sentence?

Reply:
Perhaps `Any attributes defined within a targets `templateAttributes` will be used to replace any attribute references in the template using the syntax `$attribute_name`.